homeassistant/components/electric_kiwi/const.py

```diff
@@ -0,0 +1,11 @@
+"""Constants for the Electric Kiwi integration."""
+
+NAME = "Electric Kiwi"
+DOMAIN = "electric_kiwi"
+DOMAIN_DATA = f"{DOMAIN}_data"
+VERSION = "0.0.1"
+ATTRIBUTION = "Data provided by the Juice Hacker API"
+ISSUE_URL = "https://github.com/mikey0000/core/issues"
```

Comment:
`NAME`, `DOMAIN_DATA`, `VERSION`, and `ISSUE_URL` are being defined but not actually used anywhere.

Reply:
Thanks for the constructive feedback, yeah not merge ready yet and appreciate the eyes over it anyway. Was hoping to get some feedback on the Oauth process mostly then tidy up. But after finding their documentation and actual function for refreshing is different I need to go off and create my own refresh process that matches theirs.